test_scripts/WebEngine/WebEngineProjection/025_OnExitApplication_RESOURCE_CONSTRAINT_multiple_apps_on_different_connections.lua

```diff
@@ -0,0 +1,47 @@
+---------------------------------------------------------------------------------------------------
+-- Proposal: https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0273-webengine-projection-mode.md
+--
+-- Description:
+-- Processing of the OnExitApplication notification with reason RESOURCE_CONSTRAINT from HMI
+--  (multiple applications on different connections)
+--
+-- Precondition:
+-- 1. SDL and HMI are started
+-- 2. App1 is registered on mobile connection 1
+-- 3. App2 is registered on web engine connection 2
+-- 4. App2 is activated
+--
+-- Steps:
+-- 1. HMI sends BC.OnExitApplication with reason: "RESOURCE_CONSTRAINT" related to App2 to SDL
+-- SDL does:
+--  - unregister App2 and send OnAppInterfaceUnregistered notification with reason: "RESOURCE_CONSTRAINT" to it
+--  - send BasicCommunication.OnAppUnregistered notification related to App2 with unexpectedDisconnect: false
+--  - close mobile connection 2
+--  - not close mobile connection 1
+--  - not send OnAppInterfaceUnregistered notification with reason: "RESOURCE_CONSTRAINT" to App1
+---------------------------------------------------------------------------------------------------
+--[[ Required Shared libraries ]]
+-- local events = require("events")
+local common = require('test_scripts/WebEngine/commonWebEngine')
+
+--[[ Local Variables ]]
+local APP_1 = 1
+local APP_2 = 2
+local DEFAULT = 1
+local WEB_ENGINE = 2
```

Comment:
```suggestion
local app_type_id_default = 1
local app_type_id_web_engine = 2
```

Reply:
@YarikMamykin I believe will be better to name **app_id_web_engine_type**

Reply:
@YarikMamykin All variables/constants is renamed using `camelCase` style (see d8172a5 )